moped-database/metadata/tables.yaml

```diff
@@ -3399,6 +3253,245 @@
         paused: true
         schedule: 0 0 * * *
         timeout: 60
+- table:
+    name: moped_proj_work_activity
+    schema: public
+  object_relationships:
+    - name: created_by_user
+      using:
+        foreign_key_constraint_on: created_by_user_id
+    - name: moped_project
+      using:
+        foreign_key_constraint_on: project_id
+    - name: moped_work_activity_status
+      using:
+        foreign_key_constraint_on: status_id
+    - name: moped_workgroup
+      using:
+        foreign_key_constraint_on: implementation_workgroup_id
+    - name: updated_by_user
+      using:
+        foreign_key_constraint_on: updated_by_user_id
+  insert_permissions:
+    - role: moped-admin
+      permission:
+        check: {}
+        set:
+          created_by_user_id: x-hasura-user-db-id
+          updated_by_user_id: x-hasura-user-db-id
```

Comment:
these `set` rules assign the user ID from the session token into these columns. note also that insert/edit permissions are disabled for created by/updated by/created at/updated at.